lerna.json

```diff
@@ -1,6 +1,6 @@
 {
   "packages": ["packages/*", "demo"],
-  "version": "0.0.1",
+  "version": "0.0.0",
```

Comment:
```suggestion
  "version": "0.0.0-dev",
```

Reply:
let's update version with separate PR, probably when adding release-to-npm mechanism